src/types/line.js

```diff
@@ -43,7 +43,7 @@ function limitLineToArea(p1, p2, area) {
 }
 
 export default class LineAnnotation extends Element {
-  intersects(x, y, epsilon = 0.001, useFinalPosition) {
+  intersects(x, y, useFinalPosition) {
```

Comment:
This could be a breaking change, if someone is using this **public** method.

To fix this in a non-breaking way, we should pass the square of borderWidth/2 as epsilon.

Reply:
done